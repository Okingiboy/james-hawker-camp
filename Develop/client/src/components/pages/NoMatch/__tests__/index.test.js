// Import dependencies
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import NoMatch from '..';

// After each test, make sure there isnt any leftover memory that could give you false results
afterEach(cleanup);

// Declare the components you are testing
describe('NoMatch component', () => {
  // verify the component is rendering properly
  it('NoMatch component renders', () => {
    render(<NoMatch />);
  });

  // compare snapshot versions of the DOM node structure
  it('matches snapshot DOM node structure', () => {
    const { asFragment } = render(<NoMatch />);
    expect(asFragment()).toMatchSnapshot();
  });
});