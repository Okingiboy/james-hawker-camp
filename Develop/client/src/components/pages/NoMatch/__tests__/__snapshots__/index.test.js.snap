// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div>
    <div
      style="height: 560px; clear: both; padding-top: 120px; text-align: center;"
    >
      <h1>
        404 Page Not Found
      </h1>
      <h1>
        <span
          aria-label="Face With Rolling Eyes Emoji"
          role="img"
        >
          ðŸ™„
        </span>
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
